napari/util/tests/test_keybindings.py

```diff
@@ -61,15 +61,15 @@ def test_bind_key():
     kb = {}
 
     # bind
-    f = lambda: 42
+    f = lambda: 42  # noqa
```

Comment:
Yeah, like @sofroniewn, I'm a little allergic to the `# noqa`'s everywhere. If there is a way to configure this to instead be `#flake8ignore` or somesuch (explicit is better than implicit), I'd be more inclined to ignore (heh) them. As it is, I think it's better to do as it says, and replace this with an explicit function definition.

Reply:
@tlambert03 I definitely prefer the more explicit `#noqa: E<whatever>` form, thanks for pointing that out. It's a strong hint about e.g. what to Google to figure out what that comment is doing.

Reply:
looked into briefly but could not find any way to change the syntax anything other than `# noqa`.... will opt for explicit error codes

Reply:
@tlambert03 You should be able to tell `flake8` to ignore `E731` to allow assignment of `lambda`s... could you give that a try? If that works, then you can drop the `# noqa`

Reply:
I already replaced it with an explicit function definition as @jni suggested above.  I don’t personally feel that expression to lambdas provides any particular additional clarity besides simply fitting on one line... so hesitant to add E731 to ignores